docs/user_guide/confidential_computing/confidential_azure_container_instances_deployment.rst

```diff
@@ -0,0 +1,196 @@
+-----------------------------------------
+
+With the resource group created, we first create one Azure Container Registry (ACR) which our built images are pushed to and pulled from.
+
+.. code-block:: bash
+
+   #!/usr/bin/env bash
+
+   resource_group=cc-prep-rsr-grp
+   location=eastus
+   reg_name=ccprepreg
+   dnl_scope=Unsecure
+
+   az group create --name $resource_group --location $location
+
+   az acr create --resource-group $resource_group \
+   --name $reg_name --sku Standard \
+   --dnl-scope $dnl_scope
+
+
+Build Docker Container Images
+--------------------------------------
+
+The container images we are going to build will include the startup kit for the NVFlare server.  Therefore, please
+obtain that set of files and copy the startup kit to one folder inside the current working directory.  In the following
+example, the server's startup kit and the Dockerfile are stored in nvflserver.eastus.azurecontainer.io
+folder and docker folder under current working folder, respectively, as shown below.
+
+.. code-block ::
+
+   $ tree -d -L 1
+   .
+   ├── docker
+   └── nvflserver.eastus.azurecontainer.io
+
+
+.. code-block:: bash
+
+   #!/usr/bin/env bash
+
+   tag=0.0.1
+   name=cc_prep
+   reg_name=ccprepreg
+   registry=${reg_name}.azurecr.io
+   nvfl_root=nvflserver.eastus.azurecontainer.io
+
+   docker build --build-arg nvfl_root=$nvfl_root -t $registry/$name:$tag -f docker/Dockerfile .
+
+.. code-block:: 
+   :caption: Dockerfile
+   FROM python:3.10
+   ARG NVFL_ROOT=nvflserver.eastus.azurecontainer.io
+   WORKDIR /workspace
+   RUN python3 -m pip install --no-cache nvflare
+   COPY $NVFL_ROOT nvflare
+
+
+Publish Container Images to Azure Container Registry
+------------------------------------------------------------
+
+After the above steps, we have one container image built and stored locally.  Now we will push it to ACR we created in step 2.  This step
+requires you to obtain an access token from ACR and to use it to login to ACR.
+
+
+.. code-block:: bash
+
+   #!/usr/bin/env bash
+
+   reg_name=ccprepreg
+   reg_token_file=reg_token.json
+
+   az acr login --name $reg_name --expose-token > $reg_token_file 
+
+   echo "ACR reg token saved to $reg_token_file"
+
+
+With the token file available, you can log in to the ACR.
+
+.. code-block:: bash
+
+   #!/usr/bin/env bash
+
+   reg_name=ccprepreg
+   registry=${reg_name}.azurecr.io
+   reg_token_file=reg_token.json
+
+   reg_token=$(jq -r .accessToken $reg_token_file)
+
+   docker login $registry -u 00000000-0000-0000-0000-000000000000 -p $reg_token
+
+Then you can push your newly built container image to the ACR.
+
+.. code-block:: bash
+
+   #!/usr/bin/env bash
+
+   tag=0.0.1
+   name=cc_prep
+   reg_name=ccprepreg
+   registry=${reg_name}.azurecr.io
+   
+   docker push $registry/$name:$tag
+   docker push $registry/skr:2.7
+
+.. note::
+
+   The skr:2.7 is built from Microsoft open source project at https://github.com/microsoft/confidential-sidecar-containers.
```

Comment:
note: did not work. 
  you need to indend the text

Reply:
I built the html and it worked.
<img width="564" height="172" alt="image" src="https://github.com/user-attachments/assets/e72d08bc-dc21-4db0-a382-8845c04a61b9" />